diesel_cli/README.md

```diff
@@ -131,13 +131,13 @@ Diesel can generate a bash completion script for itself:
 #### linux
 
 ```sh
-$ diesel bash-completion > /etc/bash_completion.d/diesel
+$ bash -c 'sudo /home/my-user-name/.cargo/bin/diesel completions bash > /etc/bash_completion.d/diesel'
```

Comment:
Probably better use something like 
```suggestion
$ diesel completions bash | sudo tee /etc/bash_completion.d/diesel'
```
or just leave of the sudo stuff because that's quite setup specific.